nuxeo-core/nuxeo-core-storage-mongodb/src/main/resources/OSGI-INF/mongodb-configuration-properties.xml

```diff
@@ -15,4 +15,13 @@
     <property name="nuxeo.mongodb.like.anchored">true</property>
   </extension>
 
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <documentation>
+      Property to disable the cursor timeout on the GC markReferencedBlobs query.
+
+      @since 2021.14
```

Comment:
Too late it is merged but in XML extension point doc, I would have set `@since 10.10HF57`,  2021.14 will be troubling in the explorer